handout/i18n/README.md

```diff
@@ -0,0 +1,3 @@
+## What is the purpose of i18n?
+
+Internationalization (frequently abbreviated to the numeronym i18n) is the process of adapting software and web applications to multiple languages, allowing their application to be used by multiple language-speaking users. By ensuring that your application supports multiple languages, you can reach a broader audience and ensure a smooth user experience for multilingual users.
```

Comment:
if you're going to mention the abbreviation it might be a good idea to explain that there are 18 characters between i-n in internationalization?

That's a bit nit picky though, but it's interesting.